osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -406,7 +406,7 @@ public abstract class DrawableRuleset : CompositeDrawable
         public abstract Playfield Playfield { get; }
 
         /// <summary>
-        /// Place to put drawables above hit objects but below UI.
+        /// Content to be placed above hitobjects. Will be affected by frame stability.
         /// </summary>
         public abstract Container Overlays { get; }
```

Comment:
I believe `Overlay` now is not needed anymore and can be removed?

Reply:
It is needed. Please check usages.